ld/armv7a7-imx6ull.ldt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Linker Template and Platform Config for ARMv7 A7 i.MX 6ULL
+ *
+ * Copyright 2021-2022 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef ARMV7A7_IMX6ULL_LDT
+#define ARMV7A7_IMX6ULL_LDT
+
+
+/* Platform specific definitions */
+#define SIZE_PAGE    0x1000
+#define SIZE_STACK   (2 * SIZE_PAGE)
+#define SIZE_HEAP    (2 * SIZE_PAGE)
+
+#define ADDR_DDR 0x80000000
+#define SIZE_DDR (128 * 1024 * 1024)
+
+#define ADDR_OCRAM 0x907000
+#define SIZE_OCRAM 0x020000
+
+
+#if defined(__LINKER__)
+
+/* Memory map setup */
+MEMORY
+{
+	m_ocram   (rwx) : ORIGIN = 0x00907000, LENGTH = 128k
```

Comment:
Not really - we only have 68K usable, lower adresses are reserved by the bootloader

Reply:
Usable of course if we take into the account we're starting from 0x00907000 - after bootloader completes memory below this address become available again

Reply:
Reused setup from there https://github.com/phoenix-rtos/plo/blob/cd80102775f353e995977aa3219657c4cfad4a3f/hal/armv7a/imx6ull/config.h#L44-L45